src/OpenLoco/Audio/Audio.cpp

```diff
@@ -875,13 +878,117 @@ namespace OpenLoco::Audio
         }
     }
 
+    static constexpr int32_t getAmbientMaxVolume(uint8_t zoom)
+    {
+        constexpr int32_t _volumes[]{ -1200, -2000, -3000, -3000 };
```

Comment:
these values are what I see in IDA, but do you have any idea why there would be a double `-3000`?

Reply:
its just an upper limit of what the maximum volume should be.